mod_modular_vanilla/hooks/entity/tactical/player.nut

```diff
@@ -22,6 +22,46 @@
 		return ::Math.rand(::Math.rand(0, 1) == 0 ? 0 : 1, 2);
 	}
 
+	// MV: Added
+	// Part of modularization of player.setStartValuesEx
+	// But can be useful on its own too. Adds new traits to the character up to _amount, properly filtering traits
+	// based on existing traits and background.
```

Comment:
```suggestion
	// Adds new traits to the character up to _amount, properly filtering traits based on existing traits and background.
```
The `But can be useful on its own too` feels out of place for such a more permanent and public facing documentation. That could just aswell be written on half the modularized functions